contracts/deploy/075_oeth_amo_upgrade.js

```diff
@@ -0,0 +1,52 @@
+const addresses = require("../utils/addresses");
+const { oethPoolLpPID } = require("../utils/constants");
+const { deploymentWithGovernanceProposal } = require("../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "075_oeth_amo_upgrade",
+    forceDeploy: false,
+    // forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: true,
+    // proposalId: "",
+  },
+  async ({ ethers, deployWithConfirmation }) => {
+    const cConvexEthMetaStrategyProxy = await ethers.getContract(
+      "ConvexEthMetaStrategyProxy"
+    );
+
+    // Deploy and set the immutable variables
+    const dConvexETHMetaStrategy = await deployWithConfirmation(
```

Comment:
ℹ️ Checked why there's no `initialize()` method all in the deployment file. Verified that the upgrade will work as expected since the already deployed contract would have the already called the `super._initialize`